Makefile

```diff
@@ -17,6 +17,21 @@
 # this program; if not, write to the Free Software Foundation, Inc., 59
 # Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 
+uname_m = $(shell uname -m)
+ARCH = $(uname_m)
+ifeq ($(ARCH),x86_64)
+else
```

Comment:
It's better to replace ifeq-else with empty IF branch by ifneq,

        ifneq ($(ARCH),x86_64)
                ERROR="architecture must be x86_64 pressent:$(ARCH)\n"
       endif

Also please use preceding tab in the branch body as in the example above